charts/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -174,6 +174,11 @@ spec:
             mountPath: /etc/datadog-agent/install_info
             {{- end }}
             readOnly: true
+{{- if eq (include "need-clusterchecks-confd" .) "true" }}
+          - name: confd
+            mountPath: /conf.d
+            readOnly: true
+{{- end }}
```

Comment:
the change that you made here are not needed. since the clustercheckrunner takes configuration from the DCA

Reply:
Removed !